src/main/resources/assets/lambdynlights/lang/it_it.json

```diff
@@ -1,18 +1,19 @@
-  "lambdynlights.menu.canvas.1": "È raccomandato disattivare le luci dinamiche delle entità con Canvas!",
-  "lambdynlights.menu.canvas.2": "Attualmente la luminosità dinamica è molto pesante con Canvas!",
-  "lambdynlights.option.light_sources.block_entities": "Entità blocco",
+  "lambdynlights.menu.canvas.1": "É consigliato disabilitare la luce dinamica con Canvas!",
+  "lambdynlights.menu.canvas.2": "La luce dinamica con Canvas diminuisce molto la performance!",
+  "lambdynlights.menu.tabs.general": "Generale",
+  "lambdynlights.option.light_sources.block_entities": "Block Entities",
   "lambdynlights.option.light_sources.entities": "Entità",
-  "lambdynlights.option.light_sources.water_sensitive_check": "Controllo della sensibilità dell'acqua",
-  "lambdynlights.option.mode": "Luci dinamiche",
-  "lambdynlights.tooltip.block_entities": "Attiva l'illuminazione dinamica delle entità blocco.",
-  "lambdynlights.tooltip.creeper_lighting": "Imposta la modalità d'illuminazione dinamica dei Creeper.\n- %s disattiva l'illuminazione dinamica dei Creeper.\n- %s imposta una luminosità costante.\n- %s imposta una luminosità dinamica.",
-  "lambdynlights.tooltip.entities": "Attiva l'illuminazione dinamica delle entità. Nota: l'illuminazione dinamica dei giocatori è sempre attiva.",
-  "lambdynlights.tooltip.mode.1": "Attiva le luci dinamiche. Se attivo, per esempio, un giocatore con una torcia in mano emetterà luce.",
-  "lambdynlights.tooltip.mode.2": "%s e %s avranno un leggero ritardo ma minimizzeranno gli aggiornamenti della luce.",
-  "lambdynlights.tooltip.mode.3": "%s attiva l'illuminazione dinamica soffusa.",
-  "lambdynlights.tooltip.tnt_lighting": "Imposta la modalità d'illuminazione dinamica della TNT.\n- %s disattiva l'illuminazione dinamica della TNT.\n- %s imposta una luminosità costante.\n- %s imposta una luminosità dinamica.",
-  "lambdynlights.tooltip.water_sensitive": "Attiva il controllo delle fonti di luce sensibili all'acqua. Questo vuol dire che certi oggetti non emetteranno luce mentre sommersi."
-}
\ No newline at end of file
+  "lambdynlights.option.light_sources.water_sensitive_check": "Controllo della sensibilità in acqua",
+  "lambdynlights.option.mode": "Luci Dinamiche",
+  "lambdynlights.tooltip.block_entities": "Abilita le luci dinamiche dei block entities",
+  "lambdynlights.tooltip.creeper_lighting": "Imposta la modalità di luce dinamica del Creeper.\\n- %s disabilita la luce dinamica del Creeper.\\n- %s imposta la luce dinamica ad un livello constante.\\n- %s imposta un livello di illuminazione dinamico.",
+  "lambdynlights.tooltip.entities": "Attiva la luce dinamica delle entità. Nota: i giocatori sono sempre attivi.",
+  "lambdynlights.tooltip.mode.1": "Attiva le luci dinamiche. Se un giocatore ha una torcia in mano illuminerà l'area per esempio.",
+  "lambdynlights.tooltip.mode.2": "%s e %s laggeranno un po' ma minimizzano gli update di luce.",
+  "lambdynlights.tooltip.mode.3": "%s è lo smooth lighting dinamico.",
```

Comment:
Vanilla translates smooth lightning as "Illuminazione soffusa", I see no reason to use the english words here.